.github/workflows/main.yaml

```diff
@@ -18,16 +18,45 @@ on:
           - peak
 env:
   LOAD_SIMULATION_TYPE: ${{ github.event.inputs.TRAFFIC_LOAD || 'standard' }}
+  PATIENT_COUNT: ${{ '10' }}
 jobs:
   performance-test:
     runs-on: ubuntu-latest
     outputs:
       epoch: ${{ steps.getEpoch.outputs.epoch }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Test Data Generator
+        uses: actions/checkout@v3
+        with:
+          repository: Bahmni/bahmni-test-data-generator
+          ref: BAH-2104
+          path: test-data-generator
+      - name: Generate Test Data
+        working-directory: test-data-generator/
+        run: |
+          if [ ${{env.LOAD_SIMULATION_TYPE}} == 'peak' ]
+          then
+            export PATIENT_COUNT='300'
+          elif [ ${{env.LOAD_SIMULATION_TYPE}} == 'high' ]
+          then
+            export PATIENT_COUNT='200'
+          else
+            export PATIENT_COUNT=${{env.PATIENT_COUNT}}
+          fi
+          ./gradlew run
+      - name: Checkout Performance Test Suite
+        uses: actions/checkout@v2
+        with:
+          repository: Bahmni/performance-test
+          path: performance
+      - name: Copy registration csv to Performance Test Suite
+        run: |
+          cp -r test-data-generator/output/registrations.csv performance/src/gatling/resources
       - name: Build
+        working-directory: performance/
```

Comment:
you dont need to checkout the performance repo in performance directory -that way you can remove the working-directory from each step

Reply:
removed working directory